client/me/purchases/manage-purchase/index.jsx

```diff
@@ -310,6 +311,14 @@ class ManagePurchase extends Component {
 			);
 		}
 
+		if ( isJetpackBackup( purchase ) ) {
+			return (
+				<div className="manage-purchase__plan-icon">
+					<Gridicon icon="cloud-upload" size={ 48 } />
```

Comment:
Let's confirm with @Automattic/jetpack-design that this is the right icon we want to use.

Reply:
/cc @crunnells @eeeeevon13

Reply:
I'd love it if this icon can be green. What do you think?